### DISCUSSION TITLE
Order related user emails issues

### DISCUSSION THREAD

**Discussion by Anubiella** [saleor/saleor#8038] [Q&A]
Good morning everyone!

**What I am trying to achieve**
Check and see if order-related user emails (_Order #{xxx} details_ or _Order {xxx} payment details_) are working or not.

**Description**
Basically we're working with Saleor 3.1.0-a.0.
If we complete the checkout for a product, what we're getting as response is just ONE email of the two, and if we take
a look to the order detail dashboard: 

![Order -333 - example](https://user-images.githubusercontent.com/31857092/132181782-cd40fb11-edf0-4fd8-a18a-1b01ed0ba50e.png)

Although is written in italian, I can summerize that Saleor has: sent the order, captured the payment, was entirely paid, 
was confirmed and that the email with placed information was sent to customer.
So the customer is receiving just the order details, as we can see:

![order details example](https://user-images.githubusercontent.com/31857092/132183132-608c50ef-ce5e-488d-8767-a506485c16e5.png)

As we see in the saleor dashboard, into User emails plugin page, both Order details and Payment confirmation emails
have the usual DEFAULT template:
![user email order details](https://user-images.githubusercontent.com/31857092/132183627-0614ed51-9fbe-4028-b7a7-a54f119289bb.png)
![user email payment confirmation](https://user-images.githubusercontent.com/31857092/132183632-d0846a97-cbc8-4082-88f1-34341d9a4cbe.png)

But still, just one arrives and is triggered by the right event.
Take into account that previously (I mean until the end of July),  we're receiving both emails, order and payment.

Do you have any suggestion in how and/or where to search for any caveat?
Thanks in advance,
Lorenzo

**Comment by karolkielecki** [comment]
@Anubiella Hi! Thanks for reporting. We've rolled out 3.1.0-a.3 -- could you please check it out with the newest release? If the issue appears, please let us know.

**Comment by Anubiella** [comment]
God bless, thanks a lot, I'll let you know asap.

Stay tuned :)
